modules/search/pages/default-analyzers-reference.adoc

```diff
@@ -0,0 +1,93 @@
+= Default Analyzers 
+:page-topic-type: reference
+:description: Use an analyzer to filter and modify search strings to improve matches for search results. 
+ 
+
+[abstract]
+{description}
+
+Analyzers contain:
+
+* xref:customize-index.adoc#character-filters[Character filters], which remove unwanted characters from search input.
+* xref:customize-index.adoc#tokenizers[Tokenizers], which separate input strings into individual tokens.
+* xref:customize-index.adoc#token-filters[Token filters], which modify tokens.
+
+When you xref:create-type-mapping.adoc[create a type mapping], you can choose a default analyzer for your type mappings, or xref:create-custom-analyzer.adoc[create your own].
+
+The following default analyzer options are available: 
+
+[cols="1,2"]
+|====
+|Analyzer |Description 
+
+|inherit | If you set an analyzer to `inherit`, the Search index component inherits the xref:set-advanced-settings.adoc#default-analyzer[default analyzer set for an index]. 
+
+|ar |The `ar` analyzer uses character filters, tokenizers, and token filters designed for Arabic language searches.
+
+|cjk |The `cjk` analyzer uses character filters, tokenizers, and token filters designed for Chinese, Japanese and Korean language searches.
+
+|ckb |The `ckb` analyzer uses character filters, tokenizers, and token filters designed for Kurdish language searches.
+
+|da |The `da` analyzer uses character filters, tokenizers, and token filters designed for Danish language searches.
+
+|de |The `de` analyzer uses character filters, tokenizers, and token filters designed for German language searches.
+
+|en |The `en` analyzer uses character filters, tokenizers, and token filters designed for English language searches.
+
+|es |The `es` analyzer uses character filters, tokenizers, and token filters designed for Castilian Spanish language searches.
+
+|fa |The `fa` analyzer uses character filters, tokenizers, and token filters designed for Persian language searches.
+
+|fi |The `fi` analyzer uses character filters, tokenizers, and token filters designed for Finnish language searches.
+
+|fr |The `fr` analyzer uses character filters, tokenizers, and token filters designed for French language searches.
+
+|he |The `he` analyzer uses character filters, tokenizers, and token filters designed for Hebrew language searches.
+
+|hi |The `hi` analyzer uses character filters, tokenizers, and token filters designed for Hindi language searches.
+
+|hr |The `hr` analyzer uses character filters, tokenizers, and token filters designed for Croatian language searches.
+
+|hu |The `hu` analyzer uses character filters, tokenizers, and token filters designed for Hungarian language searches.
+
+|it |The `it` analyzer uses character filters, tokenizers, and token filters designed for Italian language searches.
+
+|keyword a|
```

Comment:
might be clearer with the special analyzers all listed above, and then a list of the language codes separately?